iiop-openjdk/src/main/resources/subsystem-templates/iiop-openjdk.xml

```diff
@@ -3,7 +3,6 @@
 <config>
    <extension-module>org.wildfly.iiop-openjdk</extension-module>
    <subsystem xmlns="urn:jboss:domain:iiop-openjdk:1.0">
-       <orb socket-binding="iiop" ssl-socket-binding="iiop-ssl"/>
```

Comment:
Sorry, that comment was in the wrong place. In the subsystem template that is fine, removing in tests I am not sure